### PR TITLE
Add `mousedown` and `mouseup` handlers to `EventsExtension`

### PR DESCRIPTION
**Pull request by whawker**
### Description

Adds support for `mousedown` and `mouseup` events in @remirror/extension-events

```jsx
import { EventsExtension } from 'remirror/extension/events';
import { useExtension } from 'remirror/react';
const Editor = () => {
  useExtension(
    EventsExtension,
    ({ addHandler }) => {
      addHandler('mousedown', () => log('Mouse down!'));
    },
    [],
  );
};
```

### Checklist

- [x] I have read the [**contributing**](https://github.com/remirror/remirror/blob/HEAD/docs/contributing.md) document.
- [x] My code follows the code style of this project and `pnpm fix` completed successfully.
- [ ] I have updated the documentation where necessary.
- [x] New code is unit tested and all current tests pass when running `pnpm test`.

### CODE REVIEW

File: packages/@remirror/extension-events/src/events-extension.ts
Comment (whawker) [inline packages/@remirror/extension-events/src/events-extension.ts:23]: Wasn't sure on `mousedown` or `mouseDown` here.

File: packages/@remirror/extension-events/src/events-extension.ts
Comment (whawker) [inline packages/@remirror/extension-events/src/events-extension.ts:45]: Wasn't sure if the `earlyReturnValue` applies here?

Comment (ifiokjr) [reply packages/@remirror/extension-events/src/events-extension.ts:45]: It does, so your code is correct 👍

File: packages/@remirror/extension-events/src/events-extension.ts
Comment (ifiokjr) [reply packages/@remirror/extension-events/src/events-extension.ts:23]: You picked correctly 👍

**Review by ifiokjr — APPROVED** [review]
Good job on adding the tests and thanks for a well-written first PR. You're a pro at this TypeScript stuff 💯 

If you could add a `changeset` to describe what's changed that should be the final piece. The following command will give present you with the options to make this possible.

```bash
pnpm changeset
```

Happy to merge once this is added and the build is passing 👍

**Review by ifiokjr — CHANGES_REQUESTED** [review]
Once you add the changeset, I can merge 👍

**Comment by ifiokjr** [comment]
By the way, you can ignore the failing labeller. I'm not sure why it's failing, but it has nothing to do with your code.

**Comment by whawker** [comment]
> Once you add the changeset, I can merge

Force pushed with changeset fixup

File: packages/@remirror/extension-events/src/events-extension.ts
Comment (ifiokjr) [reply packages/@remirror/extension-events/src/events-extension.ts:45]: FYI `earlyReturnValue` means that if any of the registered handlers returns this value, it doesn't run any of the lower priority handlers. It can be set to be a predicate function as well.

**Pull request merged**
